test/common.py

```diff
@@ -727,6 +727,37 @@ def measure_wasm_code_lines(self, wasm):
     non_data_lines = [line for line in wat_lines if '(data ' not in line]
     return len(non_data_lines)
 
+  def clean_js_output(self, output):
+    """Cleaup the JS output prior to running verification steps on it.
+
+    Due to minification, when we get a crash report from JS it can sometimes
+    contina the entire program in the output (since the entire program is
+    on a single line).  In this case we can sometimes get false positives
+    when checking for strings in the output.  To avoid this false positives
+    and the make the output easier to read in such cases we attempt to removes
+    such lines from the JS output.
+    """
+    lines = output.splitlines()
+    long_lines = []
+
+    def cleanup(line):
+      if len(line) > 2048:
+        # Sanity check that this is really the emscripten program/module on
+        # a single line.
+        assert line.startswith('var Module=typeof Module!="undefined"')
```

Comment:
closure etc. changes this, doesn't it?

Reply:
If it does then this assert would catch it right away..  I've limited this cleanup to JS runs that return non-zero..